NN_start.js

```diff
@@ -28,18 +28,56 @@ nerthus.addon.consts.VERSION_URL = nerthus.addon.fileMasterUrl("version.json")
 nerthus.addon.store = function()
 {
     if(NerthusAddonUtils.storage())
-        NerthusAddonUtils.storage().nerthus = NerthusAddonUtils.parser.stringify(nerthus)
+        NerthusAddonUtils.storage()["nerthus"] = NerthusAddonUtils.parser.stringify(nerthus)
 }
 
 NerthusAddonUtils = (function()
 {
-    var call = function(func)
+    let call = function(func)
     {
         if(typeof func === 'function')
             func()
     }
 
-    var utils = {}
+    let utils = {}
+    utils.log = function (message)
+    {
+        log(message)
+    }
+    utils.log_ni = function (message)
+    {
+        log(message)
+        let $notif = $("#consoleNotif")
+        if ($notif)
+            $notif.remove()
+        if ($._data(document.querySelector(".console-input"), "events"))
+        {
+            let handler = $._data(document.querySelector(".console-input"), "events").keydown[0].handler
+            if (handler)
+            {
+                $(".console-content div:last-child").addClass("deleteNextLog")
+                let lastChild = $(".console-content div i")
+                let $input = document.createElement("input")
+                $input.value = "self.close()"
+                handler.call($input, {keyCode: 13})
+                lastChild.remove()
+            }
+        }
+    }
+    utils.muteNiConsole = function ()
```

Comment:
nie wiem co robi to muteNiConsole ale wygląda na jakiś strasznego i haka, do tego, z tego co widziałem nie jest to testowane a powiem szczerze, że nalegałbym, żeby start miał pokrycie 100%

Może zmienić logowanie na ni na konsole przeglądarki jak normalne logowanie wymaga czegoś takiego?

Reply:
Jest strasznym hakiem i nie da się zrobić tego inaczej by działało tak dobrze jak działa teraz. Chyba że mam włamywać się do `require` z silnika gry przy pomocy luki w postaci `eval` w jednej funkcji. 

Logowanie w konsoli gry jest bardzo przydatne podczas grania zwykłym osobom, więc lepiej by było by znajdowało się w konsoli gry. Szczególnie że w konsoli przeglądarki NI bardzo lubi wywalać [milion errorów](https://i.imgur.com/uY0Xr6e.png)